src/Refactoring-Core/RBPullUpMethodRefactoring.class.st

```diff
@@ -278,6 +294,21 @@ RBPullUpMethodRefactoring >> removePulledUpMethods [
 	selectors do: [:each | class removeMethod: each]
 ]
 
+{ #category : #preconditions }
+RBPullUpMethodRefactoring >> requestSuperClass [
+	^(self options at: #superClass) value: self
```

Comment:
Why #options logic is required?  
Users of refactoring can create instance with all required parameters. I always found this #options approach super bad style.